samples/server/petstore/jaxrs-spec-required-and-readonly-property/src/gen/java/org/openapitools/model/ReadonlyAndRequiredProperties.java

```diff
@@ -0,0 +1,200 @@
+package org.openapitools.model;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.io.Serializable;
+import javax.validation.constraints.*;
+import javax.validation.Valid;
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+
+
+@JsonTypeName("ReadonlyAndRequiredProperties")
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen")
+public class ReadonlyAndRequiredProperties  implements Serializable {
+  
+  private @Valid String requiredYesReadonlyYes;
+  private @Valid String requiredYesReadonlyNo;
+  private @Valid String requiredNoReadonlyYes;
+  private @Valid String requiredNoReadonlyNo;
+
+  protected ReadonlyAndRequiredProperties(ReadonlyAndRequiredPropertiesBuilder<?, ?> b) {
+  this.requiredYesReadonlyYes = b.requiredYesReadonlyYes;this.requiredYesReadonlyNo = b.requiredYesReadonlyNo;this.requiredNoReadonlyYes = b.requiredNoReadonlyYes;this.requiredNoReadonlyNo = b.requiredNoReadonlyNo;
+  }
+
+  public ReadonlyAndRequiredProperties() { }
+
+  /**
+   **/
+  public ReadonlyAndRequiredProperties requiredYesReadonlyYes(String requiredYesReadonlyYes) {
+    this.requiredYesReadonlyYes = requiredYesReadonlyYes;
+    return this;
+  }
+
+  
+  @ApiModelProperty(required = true, value = "")
+  @JsonProperty("requiredYesReadonlyYes")
```

Comment:
The change is now, that the @NotNull is not present anymore when generating the server stub for a property which is marked as readonly and required